paddle/fluid/framework/init.cc

```diff
@@ -62,6 +62,18 @@ void InitDevices() {
 
   for (int i = 0; i < count; ++i) {
     places.emplace_back(platform::CUDAPlace(i));
+    for (int j = 0; j < count; ++j) {
+      if (i == j) continue;
+      int can_acess = -1;
+      PADDLE_ENFORCE(cudaDeviceCanAccessPeer(&can_acess, i, j),
+                     "Failed to test P2P access.");
+      if (can_acess != 1) {
+        VLOG(1) << "Cannot enable P2P access from " << i << " to " << j;
```

Comment:
It may be better to use `LOG(WARNING)`.

Reply:
Agreed. VLOG for debugging only.  https://github.com/PaddlePaddle/Paddle/pull/5181
LOG(WARNING) is better here.

Reply:
Done